bokehjs/examples/stocks/stocks.ts

```diff
@@ -0,0 +1,73 @@
+console.log(`Bokeh ${Bokeh.version}`);
+Bokeh.set_log_level("debug");
+
+
+function make_plot(title:string, source:Bokeh.ColumnDataSource) {
+    
+    // Create plot
+    var xr = new Bokeh.DataRange1d({});
+    var yr = new Bokeh.DataRange1d({});
+    var plot = new Bokeh.Plot({        
```

Comment:
I strongly suggest against `var`. Preferably use always `const` and `let` for truly modifiable (!= mutable, because an array can be declared `const` but you can still mutate its internal state) variables (e.g. `i`).